dist/infragram.js

```diff
@@ -395,12 +396,12 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
     // http://github.com/p-v-o-s/infragram-js.
     module.exports = function Camera(options) {
       var canvas, ctx; // Initialize getUserMedia with options
-
       function initialize() {
-        getUserMedia(webRtcOptions, success, deviceError); // iOS Safari 11 compatibility: https://github.com/webrtc/adapter/issues/685
-
-        webRtcOptions.videoEl.setAttribute('autoplay', 'autoplay');
-        webRtcOptions.videoEl.setAttribute('playsinline', 'playsinline');
+        getUserMedia(webRtcOptions, success, deviceError); // iOS Safari 11 compatibility: https://github.com/webrtc/adapter/issues/685         
+        webRtcOptions.videoEl.setAttribute('id', 'webCamVideoEl');
```

Comment:
These changes can be made in this file: https://github.com/publiclab/infragram/blob/3f5f87aefef84d92f87eed3e4b22a0cd6edc8bf0/src/io/camera.js#L13